android/src/main/kotlin/com/chargebee/flutter/sdk/ChargebeeFlutterSdkPlugin.kt

```diff
@@ -141,6 +145,28 @@ class ChargebeeFlutterSdkPlugin : FlutterPlugin, MethodCallHandler, ActivityAwar
             })
     }
 
+
+    private fun restorePurchases(resultCallback: Result, queryParams: Map<String, Boolean>?) {
+        val includeInActivePurchases = queryParams?.get("includeInActivePurchase") as Boolean
+        CBPurchase.restorePurchases(
+            activity,
+            includeInActivePurchases,
+            object : CBCallback.RestorePurchaseCallback {
+                override fun onSuccess(result: List<CBRestoreSubscription>) {
+                    restoreSubscription.clear()
+                    result.forEach { subscription ->
+                        val jsonMapString = Gson().toJson(subscription.toMap())
+                        restoreSubscription.add(jsonMapString)
+                    }
+                    resultCallback.success(restoreSubscription)
```

Comment:
Do we need to keep `restoreSubscription` at instance level? Can we refactor to have it at method level?

Reply:
Done